sdk/nodejs/src/index.ts

```diff
@@ -143,3 +143,5 @@ export function initializeDevCycle(
  * @deprecated Use initializeDevCycle instead
  */
 export const initialize = initializeDevCycle
+
+export * from './open-feature-provider/strategy'
```

Comment:
exporting it this way means the multi provider isnt optional, since this will be evaluated any time you import anything from the main package entrypoint.
For it to be truly optional, you need to make a different entrypoint to import the strategy from, like 

`import { DevCycleStrategy } from '@devcycle/nodejs/strategy`
or 
`import { DevCycleStrategy } from '@devcycle/nodejs/openfeature`